### PR TITLE
LSR-932 Detailed error messages (iOS)

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by victorg1991** [comment]
LGTM 👍

**Comment by nhpatt** [comment]
👍

File: android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java [hunk 1]
```diff
@@ -71,7 +72,11 @@ public static Session createSessionFromCurrentSession() {
 	}
 
 	public static boolean isLoggedIn() {
-		return currentUserSession != null;
+		if (currentUserSession == null) {
+			LiferayLogger.e("You need to be logged to view the content!");
```
Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java]: Why showing an error in the console? This method just checks whether there's a session, but it's not an error not having a session

Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java]: In most of the cases this method is used when the app is going to load some content or is going to move you to the next activity and it's necessary to be logged in. It's an info message for the user.
Maybe it's better a LiferayLogger info message with: LiferayLogger.i(String).

**Pull request merged**